app/views/includes/patient-survey.nunjucks

```diff
@@ -1,11 +1,11 @@
 <h2 data-title="Patients Say">What patients say about this surgery</h2>
-<p>Every year, the GP Patient Survey gives patients a chance to say how they feel about their GP surgery.</p>
-<p>The survey is independent and asks people about their experiences of:</p>
+<p>You can see:</p>
+
 <ul>
-  <li>booking an appointment</li>
-  <li>waiting times</li>
-  <li>staff at the surgery</li>
+  <li><a class="gp-ratings-reviews" href="{{gp.choicesReviewsLink}}">ratings and reviews</a> for this surgery</li>
+  <li><a class="gp-patient-survey" href="https://gp-patient.co.uk/report?practicecode={{gp.odsCode}}">GP Patient Survey results</a> for this surgery</li>
```

Comment:
For the sake of consistency it would be good to have the entire href be done in the same fashion as others i.e. created in the app code and then the view can just render it.